wiki/People/Nomination_Assessment_Team/en.md

```diff
@@ -17,61 +17,61 @@ Members of the NAT are distinguished by their red username and user title. They
 
 ## Responsibilities
 
-The NAT is responsible for a variety of mapping-related tasks, with each NAT member falling into one of two subgroups: **evaluators** and **advisors**. Each subgroup outlines the responsibilities of its members.
+The NAT is responsible for a variety of mapping-related tasks, which are divided into two categories: **evaluation** and **structural**. Each category outlines the responsibilities of a NAT member, as well as the requirements and expectations for each type of task.
```

Comment:
> ...two categories: evaluation and structural

I'm concerned that the first one is a noun, and the second one is an adjective -- doesn't read well at all

Reply:
think of it as "the evaluation category" and "the structural category", i'm aware it doesn't sound the best but we struggled with finding good alternatives to "structural" that explain the nature of the task well